### PR TITLE
Updated Credits for PoP

### PR DESCRIPTION
**Pull request by GMPranav**
- [x] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself.
- [x] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [x] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -14654,9 +14654,12 @@
         </Games>
         <URLs>
             <URL>https://raw.githubusercontent.com/PoPRuns/PoP.AutoSplitters/main/multipop.asl</URL>
+            <URL>https://raw.githubusercontent.com/PoPRuns/PoP.AutoSplitters/main/pop_sot.asl</URL>
+            <URL>https://raw.githubusercontent.com/PoPRuns/PoP.AutoSplitters/main/pop_ww.asl</URL>
+            <URL>https://raw.githubusercontent.com/PoPRuns/PoP.AutoSplitters/main/pop_t2t.asl</URL>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:14659]: This change doesn't make sense. Why are you downloading multiple asl files for one autosplitter?

Comment (just-ero) [reply LiveSplit.AutoSplitters.xml:14659]: I strongly disagree with that approach.

Comment (just-ero) [reply LiveSplit.AutoSplitters.xml:14659]: Is it purely for the settings?

Comment (GMPranav) [reply LiveSplit.AutoSplitters.xml:14659]: Well, it avoids having to maintain splits in two places. And for such a rarely touch category extension, it's good enough.

Comment (just-ero) [reply LiveSplit.AutoSplitters.xml:14659]: You could offload the settings to an entirely separate file which is loaded by both the individual ASL and the multi-ASL.  
I see a flaw in your approach in that when the runner changes any settings in the multi-ASL while one of the games is running, this change is not translated over to the ASL on the layout.

Comment (Pranav-FP) [reply LiveSplit.AutoSplitters.xml:14659]: I could move the settings to another file, but then the splits logic would still have to be written in multipop.asl again. And updated if it's updated in the individual games.

Yes I am aware of this limitation. The solution is to set settings before starting a run.

Comment (GMPranav) [reply LiveSplit.AutoSplitters.xml:14659]: I could move the settings to another file, but then the splits logic would still have to be written in multipop.asl again. And updated if it's updated in the individual games.

Yes I am aware of this limitation. The solution is to set settings before starting a run, I have made the runners aware of it too.

Comment (just-ero) [reply LiveSplit.AutoSplitters.xml:14659]: You are also removing the ASL component when the timer resets. I'm not sure that's appropriate.

Comment (just-ero) [reply LiveSplit.AutoSplitters.xml]: Feel free to try again another time. I think the idea is fine, the implementation just needs some work.

**Pull request merged**